File: AMS_资讯信息组件/code/server/source/com.yss.ams.sec.information.support/src/main/java/com/yss/ams/sec/information/support/modules/sv/fiincome/pojo/FiIncome.java
package com.yss.ams.sec.information.support.modules.sv.fiincome.pojo;

import com.yss.framework.api.common.co.AuditableParamPojo;

/**
 * 债券每日利息pojo类
 * @author yuankai
 * 资讯信息拆分	2017.7.3 STORY #42948 资讯信息管理组件化拆分
 *
 */
public class FiIncome extends AuditableParamPojo {

	/**
	 * 证券代码
	 */
	private String c_SEC_CODE = "";

	/**
	 * 计息日期
	 */
	private String d_INCOME = "";

	/**
	 * 报价方式
	 */
	private String c_DV_QUT_MOD = "";

	/**
	 * 计息天数
	 */
	private String n_INCOME_DAYS = "";

	/**
	 * 票面利率
	 */
	private String n_COUP_RATE = "";

	/**
	 * 剩余本金
	 */
	private String n_REM_COR = "";

	/**
	 * 税前计息利息
	 */
	private String n_INCOME_PT = "";

	/**
	 * 税后计息利息
	 */
	private String n_INCOME_AT = "";

	/**
	 * 税后付息利息
	 */
	private String n_INCOME_PT_DUE = "";

	public String getC_SEC_CODE() {
		return c_SEC_CODE;
	}

	public void setC_SEC_CODE(String cSECCODE) {
		c_SEC_CODE = cSECCODE;
	}

	public String getD_INCOME() {
		return d_INCOME;
	}

	public void setD_INCOME(String dINCOME) {
		d_INCOME = dINCOME;
	}

	public String getC_DV_QUT_MOD() {
		return c_DV_QUT_MOD;
	}

	public void setC_DV_QUT_MOD(String cDVQUTMOD) {
		c_DV_QUT_MOD = cDVQUTMOD;
	}

	public String getN_INCOME_DAYS() {
		return n_INCOME_DAYS;
	}

	public void setN_INCOME_DAYS(String nINCOMEDAYS) {
		n_INCOME_DAYS = nINCOMEDAYS;
	}

	public String getN_COUP_RATE() {
		return n_COUP_RATE;
	}

	public void setN_COUP_RATE(String nCOUPRATE) {
		n_COUP_RATE = nCOUPRATE;
	}

	public String getN_REM_COR() {
		return n_REM_COR;
	}

	public void setN_REM_COR(String nREMCOR) {
		n_REM_COR = nREMCOR;
	}

	public String getN_INCOME_PT() {
		return n_INCOME_PT;
	}

	public void setN_INCOME_PT(String nINCOMEPT) {
		n_INCOME_PT = nINCOMEPT;
	}

	public String getN_INCOME_AT() {
		return n_INCOME_AT;
	}

	public void setN_INCOME_AT(String nINCOMEAT) {
		n_INCOME_AT = nINCOMEAT;
	}

	public String getN_INCOME_PT_DUE() {
		return n_INCOME_PT_DUE;
	}

	public void setN_INCOME_PT_DUE(String nINCOMEPTDUE) {
		n_INCOME_PT_DUE = nINCOMEPTDUE;
	}

	public String getN_INCOME_AT_DUE() {
		return n_INCOME_AT_DUE;
	}

	public void setN_INCOME_AT_DUE(String nINCOMEATDUE) {
		n_INCOME_AT_DUE = nINCOMEATDUE;
	}

	public String getC_DESC() {
		return c_DESC;
	}

	public void setC_DESC(String cDESC) {
		c_DESC = cDESC;
	}

	public static long getSerialversionuid() {
		return serialVersionUID;
	}

	/**
	 * 税后付息利息
	 */
	private String n_INCOME_AT_DUE = "";
	/**
	 * 描述
	 */
	private String c_DESC = "";

	private static final long serialVersionUID = 1L;
}

File: AMS_资讯信息组件/code/client/source/YssSecInformation.Support/Sv/Pojo/FiIncome.cs
using FAST.Core.Util;
using FAST.Common.Service.Pojo;
using FAST.Common.Service.Pojo.Base;
using FAST.Core.Exceptions;


using FAST.Core.Context;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

using Newtonsoft.Json;

namespace YssSecInformation.Support.Sv.Pojo
{
    /// <summary>
    /// 债券每日利息Pojo类
    /// added by ll 20120928
    /// </summary>
    public class FiIncome : EnclosurePojo
    {
        /// <summary>
        /// 证券代码
        /// </summary>
        private string c_SEC_CODE = "";

        /// <summary>
        /// 计息日期
        /// </summary>
        private string d_INCOME = "";

        /// <summary>
        /// 报价方式
        /// </summary>
        private string c_DV_QUT_MOD = "";

        /// <summary>
        /// 计息天数
        /// </summary>
        private string n_INCOME_DAYS = "";

        /// <summary>
        /// 票面利率
        /// </summary>
        private string n_COUP_RATE = "";

        /// <summary>
        /// 剩余本金
        /// </summary>
        private string n_REM_COR = "";

        /// <summary>
        /// 税前计息利息
        /// </summary>
        private string n_INCOME_PT = "";

        /// <summary>
        /// 税后计息利息
        /// </summary>
        private string n_INCOME_AT = "";

        /// <summary>
        /// 税后付息利息
        /// </summary>
        private string n_INCOME_PT_DUE = "";

        /// <summary>
        /// 税后付息利息
        /// </summary>
        private string n_INCOME_AT_DUE = "";

        /// <summary>
        /// 描述
        /// </summary>
        private string c_DESC = "";

        /// <summary>
        /// 属性: 证券代码 
        /// </summary>
        [JsonProperty(PropertyName = "c_SEC_CODE")]
        public string C_SEC_CODE
        {
            set { c_SEC_CODE = value; }

            get { return c_SEC_CODE; }
        }

        /// <summary>
        /// 属性: 计息日期 
        /// </summary>
        [JsonProperty(PropertyName = "d_INCOME")]
        public string D_INCOME
        {
            set { d_INCOME = value; }

            get { return d_INCOME; }
        }

        /// <summary>
        /// 属性: 报价方式 
        /// </summary>
        [JsonProperty(PropertyName = "c_DV_QUT_MOD")]
        public string C_DV_QUT_MOD
        {
            set { c_DV_QUT_MOD = value; }

            get { return c_DV_QUT_MOD; }
        }

        /// <summary>
        /// 属性: 计息天数 
        /// </summary>
        [JsonProperty(PropertyName = "n_INCOME_DAYS")]
        public string N_INCOME_DAYS
        {
            set { n_INCOME_DAYS = value; }

            get { return n_INCOME_DAYS; }
        }

        /// <summary>
        /// 属性: 票面利率 
        /// </summary>
        [JsonProperty(PropertyName = "n_COUP_RATE")]
        public string N_COUP_RATE
        {
            set { n_COUP_RATE = value; }

            get { return n_COUP_RATE; }
        }

        /// <summary>
        /// 属性: 剩余本金 
        /// </summary>
        [JsonProperty(PropertyName = "n_REM_COR")]
        public string N_REM_COR
        {
            set { n_REM_COR = value; }

            get { return n_REM_COR; }
        }

        /// <summary>
        /// 属性: 税前计息利息 
        /// </summary>
        [JsonProperty(PropertyName = "n_INCOME_PT")]
        public string N_INCOME_PT
        {
            set { n_INCOME_PT = value; }

            get { return n_INCOME_PT; }
        }

        /// <summary>
        /// 属性: 税后计息利息 
        /// </summary>
        [JsonProperty(PropertyName = "n_INCOME_AT")]
        public string N_INCOME_AT
        {
            set { n_INCOME_AT = value; }

            get { return n_INCOME_AT; }
        }

        /// <summary>
        /// 属性: 税后付息利息 
        /// </summary>
        [JsonProperty(PropertyName = "n_INCOME_PT_DUE")]
        public string N_INCOME_PT_DUE
        {
            set { n_INCOME_PT_DUE = value; }

            get { return n_INCOME_PT_DUE; }
        }

        /// <summary>
        /// 属性: 税后付息利息 
        /// </summary>
        [JsonProperty(PropertyName = "n_INCOME_AT_DUE")]
        public string N_INCOME_AT_DUE
        {
            set { n_INCOME_AT_DUE = value; }

            get { return n_INCOME_AT_DUE; }
        }

        /// <summary>
        /// 属性: 描述 
        /// </summary>
        [JsonProperty(PropertyName = "c_DESC")]
        public string C_DESC
        {
            set { c_DESC = value; }

            get { return c_DESC; }
        }


    }
}


